### PR TITLE
adds the documentation for the minion board to the website

### PR DESCRIPTION
**Pull request by AaronSoliz**

### CODE REVIEW

File: Docs/source/Hardware/Boards.rst [hunk 1]
```diff
@@ -439,3 +439,183 @@ and the mux switches between all of them.
...
+Context
+^^^^^^^
+**Location of the Board:** Battery box 
+
+**List of I/O and Connections:**
+    
+    * Leaderboard Input
+        * Input from BPS Leader Board, tells the Minion board when to gather measurments form the batteries. 
+    * LeaderBoard Output 
+        * Outputs to the leaderboard, transmits voltage and temperature readings from the battery modules.
```
Comment (ClarkPoon) [inline Docs/source/Hardware/Boards.rst:479]: Specifically, this is an Isolated SPI bus. Also, the input or output connectors can connect to the leader board or to other minion boards

File: Docs/source/Hardware/Boards.rst [hunk 2]
```diff
@@ -439,3 +439,183 @@ and the mux switches between all of them.
...
+    * Voltage Connectors 
+        * Input from the battery modules, transmit voltage from each battery module. The IC also uses this 
+          voltages to power itself. 
+    * Minion Shield Power  
+        * 5V output from the LTC6811 to the Minion Shield board. This is used to power the temperature sensors.
+    * Temperature Input
+        * Input from the Minion Shield board, transmits temperature data.
```
Comment (ClarkPoon) [inline Docs/source/Hardware/Boards.rst:588]: Isn't this information also on lines 476-486?

**Review by ClarkPoon — CHANGES_REQUESTED** [review]
I realize that you did not write the content for this and are just copying it over from somewhere else, so you may want whoever wrote the content to address the content related questions

File: Docs/source/Hardware/Boards.rst [hunk 3]
```diff
@@ -439,3 +439,183 @@ and the mux switches between all of them.
     :align: center
 
     LTC6811 Block Diagram 
+
+
+
+BPS Minion Board
```
[the remark this answers is not in the document]
Comment (AaronSoliz) [reply Docs/source/Hardware/Boards.rst:445]: okay will do

File: Docs/source/Hardware/Boards.rst [hunk 4]
```diff
@@ -439,3 +439,183 @@ and the mux switches between all of them.
...
+
+Overview
+^^^^^^^^
+`GitHub Link <https://github.com/lhr-solar/BPS-MinionPCB>`__
+
+
+
+Brief Description/Purpose:
+    The BPS Minion Board measures the voltage and temperature of the batteries and sends the data to the BPS Leader Board. 
+    There are four temperature connector minion boards. Each board can measure 12 modules and 16 temperature sensors.
```
[the remark this answers is not in the document]
Comment (AaronSoliz) [reply Docs/source/Hardware/Boards.rst:456]: I got rid of this part and used the description from line 430 for the "Brief Description/Purpose"

File: Docs/source/Hardware/Boards.rst [hunk 5]
```diff
@@ -439,3 +439,183 @@ and the mux switches between all of them.
...
+Brief Description/Purpose:
+    The BPS Minion Board measures the voltage and temperature of the batteries and sends the data to the BPS Leader Board. 
+    There are four temperature connector minion boards. Each board can measure 12 modules and 16 temperature sensors. 
+
+Pertinent Regulations
+^^^^^^^^^^^^^^^^^^^^^
+===========  ============================================== ===========================================================
+Regulation   Description of Regulation                      How Regulation is Met
+
+**8.3.A.5**  | System in which measurements are constantly  | The Minion board constantly takes voltage and temperature
```
[the remark this answers is not in the document]
Comment (AaronSoliz) [reply Docs/source/Hardware/Boards.rst:463]: oops this was my bad when I transferred it over I skipped over the word monitored

File: Docs/source/Hardware/Boards.rst (hunk 1 above)
Comment (AaronSoliz) [reply Docs/source/Hardware/Boards.rst:479]: I changed it to show that they are isolated SPI bus and that they can also connect to other minion boards

File: Docs/source/Hardware/Boards.rst (hunk 2 above)
Comment (AaronSoliz) [reply Docs/source/Hardware/Boards.rst:588]: this part is repeated because within our outline we have to show the different I/O and connections for different parts of the board but this Board only has a main so it has the same I/O and connections as above. for example if you look at the display board its spilt up into Main, can, and power distribution and each one has their own connections. for this board its completely up to you since it only has a main.

Comment (AaronSoliz) [reply Docs/source/Hardware/Boards.rst:588]: I went ahead and got rid of this part and did the same with other boards that also only have a main(at least the new ones I added)

File: Docs/source/Hardware/Boards.rst [hunk 6]
```diff
@@ -13,6 +13,130 @@ the :term:`Shunt Resistor`.
...
+            | by the protection system.        
+=========== ============================================== =================================================
+
+Context
+^^^^^^^
+**Location of the Board:** Battery Box 
+
+**List of I/O and Connections:**
+    
+    * Power +12 V
```
Comment (manthanand) [inline Docs/source/Hardware/Boards.rst:55]: Would the MOSI, MISO, clock, and chip select lines not be included here?

File: Docs/source/Hardware/Boards.rst [hunk 7]
```diff
@@ -13,6 +13,130 @@ the :term:`Shunt Resistor`.
...
+**Location of the Board:** Battery Box 
+
+**List of I/O and Connections:**
+    
+    * Power +12 V 
+        * Input from BPS Leader Board 
+    * Power GNDPWR 
+        * Input from BPS Leader Board 
+    * Signal ETS  
+        * Input from Shunt Resistor Board
```
Comment (manthanand) [inline Docs/source/Hardware/Boards.rst:60]: I guess this doesn't matter for other connections (like TX or RX), but it would be helpful to know what these acronyms mean or what they are used for

File: Docs/source/Hardware/Boards.rst [hunk 8]
```diff
@@ -427,15 +541,593 @@ the +5V input and corresponding grounds are already isolated from the +12V line.
...
+
+*What does this circuit do?*
+    This circuit measures the voltages and temperatures of each battery module 
+    and sends the data to the leader board when prompted. 
+*Why do we need it?*
+    Regulations stipulate that the voltages and temperatures of the battery 
+    modules must be monitored at all times. 
+*List of Circuit Components*
+    * 2 Molex Micro-fit 3.0 1x07 3.00mm Horizontal  
+        * Description: Each connector allows for eight connections to the battery modules
```
Comment (manthanand) [inline Docs/source/Hardware/Boards.rst:611]: seven connections?

File: Docs/source/Hardware/Boards.rst [hunk 9]
```diff
@@ -427,15 +541,593 @@ the +5V input and corresponding grounds are already isolated from the +12V line.
...
+Brief Description/Purpose:
+    The purpose of this board is to emulate the battery modules for minion board testing  
+
+Context
+^^^^^^^
+**Location of the Board:**  it's tool for testing the minion boards so it doesn’t have an exact location in the car 
+
+**List of I/O and Connections:**
+    
+    * +5V
```
Comment (manthanand) [inline Docs/source/Hardware/Boards.rst:965]: This has weird formatting when it is built. I would add "Power" in front of it because it might be due to the "+" being right after the "*".

File: Docs/source/Hardware/Boards.rst (hunk 7 above)
Comment (AaronSoliz) [reply Docs/source/Hardware/Boards.rst:60]: okay will add the acronym or what they are based off the datasheet

File: Docs/source/Hardware/Boards.rst (hunk 9 above)
Comment (AaronSoliz) [reply Docs/source/Hardware/Boards.rst:965]: okay will do

File: Docs/source/Hardware/Boards.rst (hunk 8 above)
Comment (AaronSoliz) [reply Docs/source/Hardware/Boards.rst:611]: not sure why the original doc says eight but I'm sure they meant seven

**Review by ClarkPoon — APPROVED** [review]
While the style and formatting of these docs is not ideal (I find that it can be a bit hard to follow, especially with all of the boards in one giant page, which is mainly my fault), I don't see any glaring factual errors, and the content seems to all be there. I also recognize that at the moment, we do not have the resources (in terms of active members) to make fixing the formatting of the hardware docs a priority. I am going to approve this pull request, so we can integrate its content with the rest of our docs. However, I am planning on opening an issue ticket with the intention of revising these docs when we have the resources.
